src/IonUnicode.ts

```diff
@@ -13,6 +13,11 @@
  * permissions and limitations under the License.
  */
 
+import { TextDecoder } from "util";
```

Comment:
Why is this being imported from `util`? The [current version of that file](https://github.com/amzn/ion-js/blob/be789e3f99f0c2ebd5365a634000eea62a827ddf/src/util.ts) doesn't seem to reference it. If the runtime supports it, won't it be available globally without an import?

Reply:
It is getting imported from @types/node/util.d.ts